### PR TITLE
fix: define and document MAP_BASE_URL to prevent gray maps

### PR DESCRIPTION
**Pull request by Aletor93**
### Checklist

- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing

### Description

The maps are gray when we start the application with this commit id ae16265b8041d32a45964bb0d208d8eb6cf5a347. 
A functional fix was suggested by a maintainer. My suggestion is to add it to the default environment variables template.

Fix suggestion: https://github.com/ibi-group/datatools-ui/issues/983#issuecomment-1922182079
Related issue: https://github.com/ibi-group/datatools-ui/issues/983

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
This looks great thanks for these changes!

File: configurations/default/env.yml.tmp [hunk 1]
```diff
@@ -6,6 +6,7 @@ MAP_BASE_URL: optional-map-tile-url
 MAPBOX_ACCESS_TOKEN: your-mapbox-access-token
 MAPBOX_MAP_ID: mapbox/outdoors-v11
 MAPBOX_ATTRIBUTION: <a href="https://www.mapbox.com/about/maps/" target="_blank">&copy; Mapbox &copy; OpenStreetMap</a> <a href="https://www.mapbox.com/map-feedback/" target="_blank">Improve this map</a>
+MAP_BASE_URL: http://tile.openstreetmap.org/{z}/{x}/{y}.png # Keep it until a fix or maps are gray
```
Comment (binh-dam-ibigroup) [inline configurations/default/env.yml.tmp]: The comment seems incomplete. Could you reword it, please?

**Review by binh-dam-ibigroup — APPROVED** [review]
Could you just fix the comment for the `MAP_BASE_URL` parameter, please?

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Could you just fix the comment for the MAP_BASE_URL parameter, please?

**Comment by Aletor93** [comment]
Sorry for the delay, I've commented MAP_BASE_URL.

**Review by binh-dam-ibigroup — APPROVED** [review]

**Pull request merged**